tests/publiccloud/ipa.pm

```diff
@@ -47,28 +38,34 @@ sub find_ami {
     return;
 }
 
+sub save_logs {
+    my $output = script_output("find ipa_results -type f");
+    for my $file (split(/\n/, $output)) {
```

Comment:
You should not rely on `script_output` giving you *just* the output of the command you gave. It might include more content that is *laying around* on the serial port output. Better use regex-matching here to just match you expect